openslides/core/static/js/core/site.js

```diff
@@ -525,33 +525,88 @@ angular.module('OpenSlidesApp.core.site', [
 .controller('SearchBarCtrl', [
     '$scope',
     '$state',
-    function ($scope, $state) {
-        $scope.search = function(query) {
+    '$sanitize',
+    function ($scope, $state, $sanitize) {
+        $scope.search = function() {
+            var query = $sanitize($scope.query);
```

Comment:
`$scope.query` comes from the SearchBar. Upon `search()`, we go to the SearchCtrl, and will have a different `$scope`, thus we are not able to access this `$scope.query` any more, but a *different* `$scope.query`, which will be empty if we don't arrive from the `.search` state.